### PR TITLE
error_response enhancement

### PR DESCRIPTION
**Pull request by pavan-maddula**

### CODE REVIEW

File: kwikapi/api.py [hunk 1]
```diff
@@ -455,8 +455,13 @@ def _find_request_protocol(self, request):
         return self.PROTOCOLS[protocol]
 
     def _handle_exception(self, req, e):
-        message = e.message if hasattr(e, 'message') else str(e)
-        message = '[(%s) %s: %s]' % (self.api._id, e.__class__.__name__, message)
+        if len(e.args) == 1 and type(e.args[0]) is dict:
```
Comment (prashanthellina) [inline kwikapi/api.py:458]: @pavan-maddula How will the service level code use this. Please post here an example of how an exception is raised in a service API method.

File: kwikapi/api.py [hunk 2]
```diff
@@ -508,7 +514,8 @@ def handle_request(self, request):
 
         except Exception as e:
             m = self._handle_exception(request, e)
-            response.write(dict(success=False, message=m), protocol)
+            m['success'] = False
+            response.write(m, protocol)
```
Comment (prashanthellina) [inline kwikapi/api.py:518]: @pavan-maddula  Please post an example of response format is changed based on this change

File: kwikapi/api.py (hunk 1 above)
Comment (pavan-maddula) [reply kwikapi/api.py]: Example:
raising error with a dictionary as a parameter in service level.
```
try:
        ...
except:
        raise Exception({'message':self.error_message, 'code':103})
```

File: kwikapi/api.py (hunk 2 above)
Comment (pavan-maddula) [reply kwikapi/api.py]: when raising exception user can pass a dictionary as a parameter.
Example:
`raise Exception({'message':self.error_message, 'code':103})`

API response using existing kwikapi implementation:
```
{
  "success": false,
  "message": "[(adj_hp100.servers.nferx.com:9105) Exception: {'message': \"No adjacency for '0'\", 'code': 103}]"
}
```
API response of latest implimentation:
```
{
  "code": 103,
  "message": "No adjacency for '0'",
  "success": false,
  "error": "[(adj_hp100.servers.nferx.com:9105) Exception]"
}
```

File: kwikapi/api.py (hunk 1 above)
Comment (prashanthellina) [reply kwikapi/api.py:458]: @pavan-maddula @mohansriharsha Please call me about this

**Pull request closed without merging**